examples/json-to-record/json_to_record.md

```diff
@@ -0,0 +1,9 @@
+# JSON to Record conversion
+
+The `data.jsondata` library provides multiple APIs to perform the conversion from JSON data in the form of a `string`, `byte[]`, `byte-block-stream`, and `Ballerina JSON value` to a `Ballerina record`.
+
+For more information on the underlying module, see the [`data.jsondata` module](https://lib.ballerina.io/ballerina/data.jsondata/latest/).
```

Comment:
Was it intended to use the same description for both BBEs? It's preferable for the projection BBE to have a distinct description that explains its specific use case.

Reply:
It is not exactly same, we used `selectively convert required fields from JSON data` this sentence in the projection BBE. Is not sufficient to convey the behaviour?